### ISSUE TITLE
Add rule for no-space-in-console rule

### ISSUE THREAD

**Issue by albinekb** [sindresorhus/eslint-plugin-unicorn#175]
see https://github.com/wulkano/kap/pull/450#discussion_r188170111

`console.log` (and .warn etc), all add spaces between the messages: 
```js
console.log('a', 'b', 'c') => `a b c`
```

This should be considered invalid:
```js
console.log('a ', 'b ', 'c') => `a  b  c`
```
